### DISCUSSION TITLE
Upstreaming Erigon modification

### DISCUSSION THREAD

**Discussion by META-DREAMER** [otterscan/otterscan#414] [General]
What would it take to upsteam the otterscan specific changes to Erigon so that otterscan could be used with default Erigon installations? e.g. via DappNode's Erigon package

**Comment by wmitsuda** [comment]
you are lucky, right now we are discussing that inside Erigon's discord, otterscan channel, if you are there, please feel free to join the conversation and/or watch the discussion.

Dappnode is a strong argument in favor of it, I also got this feedback from other users.

I'm going to propose a PR for it.

**Comment by wmitsuda** [comment]
Closing this discussion since all patches were already upstreamed!
